pkg/apis/pravega/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,16 +1,22 @@
 // +build !ignore_autogenerated
 
-/**
- * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- */
-
-// This file was autogenerated by deepcopy-gen. Do not edit it manually!
+/*
+Copyright The Kubernetes Authors.
```

Comment:
Have we copied this file from somewhere else? If so, why did we have to do it?

Reply:
This file is generated by the SDK based on the Custom Resource API specs. See more: https://github.com/operator-framework/operator-sdk/blob/00ef545399db0e4b8fe4474188a342369874162a/doc/sdk-cli-reference.md#generate